arch/arm/core/aarch64/prep_c.c

```diff
@@ -28,6 +29,15 @@ extern FUNC_NORETURN void z_cstart(void);
 void z_arm64_prep_c(void)
 {
 	z_bss_zero();
+
+#if !defined(CONFIG_ARM_CUSTOM_INTERRUPT_CONTROLLER)
```

Comment:
Does it make sense to move all this into a function named `z_arm_int_lib_init(void)` as we have for AARC32?
It will be a step towards aligning the internal apis

Reply:
Ok, I will do that.

Though, I think the function name `z_arm_int_lib_init` is somewhat misleading (`_lib_` should not really be there). Maybe that can be fixed in the future when we have another go at renaming things.

Reply:
Agreed. I will send a PR to change that for Cortex-M. Hold on

Reply:
Added `z_arm64_interrupt_init` function.